src/tests/topp/MzTabExporter_4_output.mzTab

```diff
@@ -6,16 +6,20 @@ MTD	description	OpenMS export from consensusXML
 MTD	protein_search_engine_score[1]	[, , Epifany Posterior Probability, 2.6.0-pre-feature-cXMLProtGroups-2020-04-06]
 MTD	peptide_search_engine_score[1]	[, , XTandem Posterior Probability, ]
 MTD	psm_search_engine_score[1]	[, , XTandem Posterior Probability, ]
-MTD	software[1]	[MS, MS:1000752, TOPP software, 2.6.0-pre-feature-MzTab-stream-2020-05-02]
-MTD	software[2]	[, , XTandem, ]
-MTD	software[2]-setting[1]	db:/Users/pfeuffer/svn/OpenMS/Tutorials/UM_2015/Example_Data/Labelfree/databases/s_pyo_sf370_potato_human_target_decoy_with_contaminants.fasta
-MTD	software[2]-setting[2]	null
-MTD	software[2]-setting[3]	null
-MTD	software[2]-setting[4]	fragment_mass_tolerance:0.1
-MTD	software[2]-setting[5]	fragment_mass_tolerance_unit:Da
-MTD	software[2]-setting[6]	precursor_mass_tolerance:5.0
-MTD	software[2]-setting[7]	precursor_mass_tolerance_unit:ppm
-MTD	software[2]-setting[8]	enzyme:Trypsin
+MTD	software[1]	[MS, MS:1000752, TOPP software, 2.6.0-pre-fix-mzTabMTD-2020-05-12]
+MTD	software[2]	[, , Epifany, 2.6.0-pre-feature-cXMLProtGroups-2020-04-06]
+MTD	software[3]	[, , XTandem, ]
+MTD	software[3]-setting[1]	db:/Users/pfeuffer/svn/OpenMS/Tutorials/UM_2015/Example_Data/Labelfree/databases/s_pyo_sf370_potato_human_target_decoy_with_contaminants.fasta
+MTD	software[3]-setting[2]	db_version:
+MTD	software[3]-setting[3]	enzyme:Trypsin
+MTD	software[3]-setting[4]	precursor_mass_tolerance:5.0
+MTD	software[3]-setting[5]	precursor_mass_tolerance_ppm:ppm
+MTD	software[3]-setting[6]	fragment_mass_tolerance:0.1
+MTD	software[3]-setting[7]	fragment_mass_tolerance_ppm:Da
+MTD	software[3]-setting[8]	charges:
+MTD	software[3]-setting[9]	missed_cleavages:1
+MTD	software[3]-setting[10]	fixed_modificiations:Carbamidomethyl (C)
+MTD	software[3]-setting[11]	variable_modificiations:Acetyl (N-term) Gln->pyro-Glu (N-term Q) Glu->pyro-Glu (N-term E)
```

Comment:
maybe a different separator?

Reply:
Instead of ":"?

Reply:
no the space character

Reply:
Ah ok. What is best for mztab? Just comma?